Nits found by various static check tools

```
$ unconvert ./...
internal/chunkedfile/chunkedfile.go:61:18: unnecessary conversion
starlark/int.go:43:42: unnecessary conversion
starlark/value.go:427:16: unnecessary conversion
$ ineffassign .
starlark/library.go:286:2: ineffectual assignment to ok
$ staticcheck  ./... |grep -v ST1006 # ignore different receiver names
internal/compile/compile.go:1:1: package comment should be of the form "Package compile ..." (ST1000)
repl/repl.go:1:1: package comment should be of the form "Package repl ..." (ST1000)
starlark/hashtable.go:284:22: func (*hashtable).dump is unused (U1000)
starlark/library.go:140:6: func builtinMethodOf is unused (U1000)
starlark/library.go:1809:6: func string_lstrip is unused (U1000)
starlark/library.go:1867:6: func string_rstrip is unused (U1000)
starlark/value.go:1100:7: identical expressions on the left and right side of the '!=' operator (SA4000)
starlark/value.go:1115:7: identical expressions on the left and right side of the '!=' operator (SA4000)
syntax/scan.go:380:2: field triple is unused (U1000)
```

I can file a PR to fix these.  Just wondering if you had any opinions on them.  For example,  the `identical expressions` are using `y != y` directly instead of `math.IsNaN()`.  Will `string_lstrip` will be used in the future or can it be removed?   The `hashtable.dump` method is probably for debugging, so I'll leave that.  Package comment tweaks are minor and inoffensive.

--- a/internal/chunkedfile/chunkedfile.go
+++ b/internal/chunkedfile/chunkedfile.go
@@ -58,7 +58,6 @@ func Read(filename string, report Reporter) (chunks []Chunk) {
 	}
 	linenum := 1
 	for i, chunk := range strings.Split(string(data), "\n---\n") {
-		chunk := string(chunk)
 		if debug {
 			fmt.Printf("chunk %d at line %d: %s\n", i, linenum, chunk)
 		}

--- a/internal/compile/compile.go
+++ b/internal/compile/compile.go
@@ -1,4 +1,4 @@
-// The compile package defines the Starlark bytecode compiler.
+// Package compile defines the Starlark bytecode compiler.
 // It is an internal package of the Starlark interpreter and is not directly accessible to clients.
 //
 // The compiler generates byte code with optional uint32 operands for a

--- a/repl/repl.go
+++ b/repl/repl.go
@@ -1,4 +1,4 @@
-// The repl package provides a read/eval/print loop for Starlark.
+// Package repl provides a read/eval/print loop for Starlark.
 //
 // It supports readline-style command editing,
 // and interrupts through Control-C.

--- a/starlark/int.go
+++ b/starlark/int.go
@@ -55,7 +55,7 @@ func MakeUint64(x uint64) Int {
 		// See comment in MakeInt64 for an explanation of this optimization.
 		return Int{new(big.Int).SetBits([]big.Word{big.Word(x)})}
 	}
-	return Int{new(big.Int).SetUint64(uint64(x))}
+	return Int{new(big.Int).SetUint64(x)}
 }
 
 var (

--- a/starlark/library.go
+++ b/starlark/library.go
@@ -137,20 +137,6 @@ var (
 	}
 )
 
-func builtinMethodOf(recv Value, name string) builtinMethod {
-	switch recv.(type) {
-	case String:
-		return stringMethods[name]
-	case *List:
-		return listMethods[name]
-	case *Dict:
-		return dictMethods[name]
-	case *Set:
-		return setMethods[name]
-	}
-	return nil
-}
-
 func builtinAttr(recv Value, name string, methods map[string]builtinMethod) (Value, error) {
 	method := methods[name]
 	if method == nil {
@@ -283,7 +269,7 @@ func UnpackPositionalArgs(fnname string, args Tuple, kwargs []Tuple, min int, va
 }
 
 func unpackOneArg(v Value, ptr interface{}) error {
-	ok := true
+	var ok bool
 	switch ptr := ptr.(type) {
 	case *Value:
 		*ptr = v
@@ -1805,14 +1791,6 @@ func string_lower(fnname string, recv Value, args Tuple, kwargs []Tuple) (Value,
 	return String(strings.ToLower(string(recv.(String)))), nil
 }
 
-// https://github.com/google/starlark-go/blob/master/doc/spec.md#string·lstrip
-func string_lstrip(fnname string, recv Value, args Tuple, kwargs []Tuple) (Value, error) {
-	if err := UnpackPositionalArgs(fnname, args, kwargs, 0); err != nil {
-		return nil, err
-	}
-	return String(strings.TrimLeftFunc(string(recv.(String)), unicode.IsSpace)), nil
-}
-
 // https://github.com/google/starlark-go/blob/master/doc/spec.md#string·partition
 func string_partition(fnname string, recv_ Value, args Tuple, kwargs []Tuple) (Value, error) {
 	recv := string(recv_.(String))
@@ -1861,14 +1839,6 @@ func string_rfind(fnname string, recv Value, args Tuple, kwargs []Tuple) (Value,
 // https://github.com/google/starlark-go/blob/master/doc/spec.md#string·rindex
 func string_rindex(fnname string, recv Value, args Tuple, kwargs []Tuple) (Value, error) {
 	return string_find_impl(fnname, string(recv.(String)), args, kwargs, false, true)
-}
-
-// https://github.com/google/starlark-go/blob/master/doc/spec.md#string·rstrip
-func string_rstrip(fnname string, recv Value, args Tuple, kwargs []Tuple) (Value, error) {
-	if err := UnpackPositionalArgs(fnname, args, kwargs, 0); err != nil {
-		return nil, err
-	}
-	return String(strings.TrimRightFunc(string(recv.(String)), unicode.IsSpace)), nil
 }
 
 // https://github.com/google/starlark-go/starlark/blob/master/doc/spec.md#string·startswith

--- a/starlark/value.go
+++ b/starlark/value.go
@@ -424,7 +424,7 @@ func (s String) Index(i int) Value     { return s[i : i+1] }
 
 func (s String) Slice(start, end, step int) Value {
 	if step == 1 {
-		return String(s[start:end])
+		return s[start:end]
 	}
 
 	sign := signum(step)

--- a/syntax/scan.go
+++ b/syntax/scan.go
@@ -377,7 +377,6 @@ type tokenValue struct {
 	float  float64  // decoded float
 	string string   // decoded string
 	pos    Position // start position of token
-	triple bool     // was string triple quoted?
 }
 
 // startToken marks the beginning of the next input token.

